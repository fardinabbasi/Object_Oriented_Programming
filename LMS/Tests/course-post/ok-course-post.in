POST login ? id 0 password UT_account
POST course_offer ? course_id 1 professor_id 810201475 capacity 60 time Saturday:14-16 exam_date 1403/4/3 class_number 5
POST logout ?
POST login ? id 810201475 password ry
POST post ? title "hi" message "hello"
POST post ? title "HW1" message "deadline: 2 days"
POST course_post ? id 1 title "next session" message "canceled"
POST logout ?
POST login ? id 810101551 password hehe
GET personal_page ? id 810201475
PUT my_courses ? id 1
GET course_channel ? id 1