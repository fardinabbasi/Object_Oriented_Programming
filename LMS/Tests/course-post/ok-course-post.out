OK
OK
OK
OK
OK
OK
OK
OK
OK
khousravi Computer Engineering Professor Advanced Programming
2 "HW1"
1 "hi"
OK
1 Advanced Programming 60 khousravi Saturday:14-16 1403/4/3 5
1 khousravi "next session"
